app/src/main/java/com/example/indoor_volleyball/CreateGymActivity.java

```diff
@@ -0,0 +1,96 @@
+package com.example.indoor_volleyball;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.example.indoor_volleyball.databinding.ActivityCreateEventBinding;
+import com.example.indoor_volleyball.databinding.ActivityCreateGymBinding;
+import com.google.android.gms.common.api.Status;
+import com.google.android.libraries.places.api.Places;
+import com.google.android.libraries.places.api.model.Place;
+import com.google.android.libraries.places.widget.Autocomplete;
+import com.google.android.libraries.places.widget.AutocompleteActivity;
+import com.google.android.libraries.places.widget.model.AutocompleteActivityMode;
+
+import java.util.Arrays;
+import java.util.List;
+
+
+public class CreateGymActivity extends AppCompatActivity {
+    ActivityCreateGymBinding binding;
+    public static final String TAG ="CreateGymActivity";
+
+    //Initialize Variables
+    EditText editText;
+    TextView textView1, textView2;
+
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        binding = ActivityCreateGymBinding.inflate(getLayoutInflater());
+
+        View view = binding.getRoot();
+
+        setContentView(view);
+
+        //Assign Variables
+        editText = findViewById(R.id.edit_text);
+        textView1 = findViewById(R.id.text_view1);
+        textView2 = findViewById(R.id.text_view2);
+        //Initialize Places
+        Places.initialize(getApplicationContext(),"AIzaSyAHKBaHxum5kdPy9qXg2vSbj57te127ScA");
```

Comment:
This is your private key and if I use it, I can screw up your API limit.  Now that it's online, it's public so anybody can take it and use it and cost you money.  Probably belongs in that secret.properties file